### ISSUE TITLE
Bandsnatch does not skip over releases with no digital downloads

### ISSUE THREAD

**Issue by betodealmeida** [Ovyerus/bandsnatch#14]
I'm trying to download my collection, but I get an error on [this album](https://tuumusic.bandcamp.com/album/invocation):

```
$ RUST_BACKTRACE=1 ./bandsnatch-linux-x86_64 run -f flac -c cookies.json -o Music redacted_username
Trying to download 1 releases
thread '<unnamed>' panicked at src/api/mod.rs:233:74:
called `Result::unwrap()` on an `Err` value: Error("missing field `downloads`", line: 1, column: 4586)
stack backtrace:
note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.
Failed to get item info for https://bandcamp.com/download?from=collection&payment_id=redacted&sig=redacted&sitem_id=218651806.
Run with `--debug` to see the full JSON blob.

Finished!
```

The redacted link takes to a page where the download link is never generated:

![Screenshot 2024-08-04 at 19-05-25 Bandcamp](https://github.com/user-attachments/assets/8ff0afd6-84fc-47e2-b6f5-904bfcea9854)

The output of `--debug` is [here](https://gist.github.com/betodealmeida/e1f2c20bfee8a3557bf8b92179bc3630).

Is there a way to skip certain albums?

**Comment by Ovyerus** [comment]
Seems like it's because this release is physical-only, which I didn't know was possible on Bandcamp! The release that actually has a digital download seems to be at https://oryxrecords.bandcamp.com/album/invocation-lp.

I'll fix Bandcamp to handle this situation.

**Comment by Ovyerus** [comment]
Fixed in 2be782ede476c2232bf7e10aeaf9f3d297966078, I'll have a release out soon.

**Comment by Ovyerus** [comment]
Released in v0.3.3
